waypoint_replanner:
  # IP and port to bind and listen for incoming velocity profiles
  server_ip: 127.0.0.1
  server_port: 1551

  # Max speed of all waypoints (km/h)
  max_speed: 60.0

  # Max Yaw rate
  max_yaw_rate: 1


  # Time Out duration (s)
  time_out: 1.0

  # Minimum time gap between two messages (s)
  time_gap: 6

  # 1m lateral transition duration (s) (Normalized to 1m)
  lateral_transition_duration: 2.0 # Based on 3m in 6s for normal behavior